repo/packages/K/kafka/0/marathon.json

```diff
@@ -0,0 +1,13 @@
+{
+  "id": "kafka",
+  "cpus": 0.5,
+  "mem": 256,
+  "instances": 1,
+  "cmd": "java -jar kafka-mesos*.jar scheduler --master=zk://master.mesos:2181/mesos --zk=master.mesos:2181 --api=http://localhost:$PORT0",
```

Comment:
Not sure what's you're trying to do, but $PORT0 is not a port you use to communicate with the master, $PORT0 is what marathon got from Mesos to assign to your framework to use. 
Why do you need to access http://master.mesos:$PORT0/?